math/vector_cross_product.cpp

```diff
@@ -0,0 +1,132 @@
+/**
+ * @file
+ *
+ * @brief Calculates the [Cross Product](https://en.wikipedia.org/wiki/Cross_product) and the magnitude of two mathematical 3D vectors.
+ *
+ *
+ * @details Cross Product of two vectors gives a vector.
+ * Direction Ratios of a vector are the numeric parts of the given vector. They are the tree parts of the
+ * vector which determine the magnitude (value) of the vector.
+ * The method of finding a cross product is the same as finding the determinant of an order 3 matrix consisting
+ * of the first row with unit vectors of magnitude 1, the second row with the direction ratios of the
+ * first vector and the third row with the direction ratios of the second vector.
+ * The magnitude of a vector is it's value expressed as a number.
+ * Let the direction ratios of the first vector, P be: a, b, c
+ * Let the direction ratios of the second vector, Q be: x, y, z
+ * Therefore the calculation for the cross product can be arranged as:
+ *
+ * ```
+ * P x Q:
+ *  	1	1	1
+ *  	a	b	c
+ *  	x	y	z
+ * ```
+ *
+ * The direction ratios (DR) are calculated as follows:
+ *  	1st DR, J:  (b * z) - (c * y)
+ *  	2nd DR, A: -((a * z) - (c * x))
+ *  	3rd DR, N:  (a * y) - (b * x)
+ *
+ * Therefore, the direction ratios of the cross product are: J, A, N
+ * The following C++ Program calculates the direction ratios of the cross products of two vector.
+ * The program uses a function, cross() for doing so.
+ * The direction ratios for the first and the second vector has to be passed one by one seperated by a space character.
+ *
+ * Magnitude of a vector is the square root of the sum of the squares of the direction ratios.
+ *
+ * @example An example of a running instance of the executable program:
```

Comment:
in doxygen, the `@example` has a special [syntax and meaning](https://www.doxygen.nl/manual/commands.html#cmdexample

Reply:
Hello @kvedala! I have made the changes. However I am not sure if I have written it correctly. Can you please check?